sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/PushDownUtils.scala

```diff
@@ -74,6 +74,34 @@ object PushDownUtils extends PredicateHelper {
     }
   }
 
+  /**
+   * Pushes down partition filters and data filters to the data source reader
+   *
+   * @return pushed partition filters.
+   */
+  def pushPartitionFilters(
```

Comment:
I think this name is a bit confusing: it doesn't **push** the partition filters but rather just set the filters in the scan builder?